.trigger-ci

```diff
@@ -1 +1 @@
-trigger-02
+23-08-2021
```

Comment:
Suggest using `2021-08-23`.

Reply:
Yes, that makes sense. I was just defaulting to European time format :)

Reply:
ISO8601 FTW!